assets/app/views/edit/build-config.html

```diff
@@ -0,0 +1,526 @@
+                          </div>
+                        </div>
+                        <div class="section">
+                          <h3>Triggers</h3>
+                          <dl class="dl-horizontal left">
+                            <div class="form-groups">
+                              <div class="checkbox" ng-if="sources.git" >
+                                <label>
+                                  <input type="checkbox" ng-model="pickedTriggers['webhook']"/>
+                                  Enable the webhook build trigger
+                                  <span class="help action-inline">
+                                    <a href>
+                                      <i class="pficon pficon-help" data-toggle="tooltip" data-placement="right" aria-hidden="true" data-original-title="The source repository must be configured to use the webhook to trigger a build when source is committed.">
+                                      </i>
+                                    </a>
+                                  </span>
+                                </label>
+                              </div>
+                              <div class="checkbox">
+                                <label>
+                                  <input type="checkbox" ng-model="pickedTriggers['imageChange']" ng-disabled="builderOptions.pickedType === 'None'"/>
+                                  Automatically build a new image when the builder image changes
+                                  <span class="help action-inline">
+                                    <a href>
+                                      <i class="pficon pficon-help" data-placement="right" aria-hidden="true"
+                                      data-toggle="tooltip" data-original-title="Automatically building a new image when the builder image changes allows your code to always run on the latest updates.">
+                                    </i>
+                                    </a>
+                                  </span>
+                                </label>
+                              </div>
+                              <div class="checkbox">
+                                <label>
+                                  <input type="checkbox" ng-model="pickedTriggers['configChange']"/>
+                                  Automatically build a new image when the build configuration changes
+                                </label>
+                              </div>
+                            </div>
+                          </dl>
+                        </div>
+                      </div>
+                    </div>
+                      <div class="buttons gutter-top-bottom gutter-top-bottom-2x">
+                        <button class="btn btn-primary btn-lg"
+                            ng-click="save()"
+                            ng-disabled="form.$invalid || disableInputs">
```

Comment:
Let's add `form.$pristine` here so the user has to change something before save is enabled.

```
ng-disabled="form.$invalid || form.$pristine || disableInputs"
```